fedimint-server/src/net/p2p.rs

```diff
@@ -343,7 +327,7 @@ impl<M: Send + 'static> P2PConnectionSMCommon<M> {
                             .with_label_values(&[self.our_id_str.as_str(), self.peer_id_str.as_str(), "incoming"])
                             .inc();
 
-                         self.incoming_sender.send(message).await.ok()?;
+                         self.incoming_sender.try_send(message).ok();
```

Comment:
At least a `debug!` when we drop messages here? Who knows what unexpected bugs this will be causing.